### PR TITLE
test(ARCH-497/module-to-cdn): fix tests and move them to cron

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

[module-to-cdn modules.json](https://unpkg.com/@talend/module-to-cdn/modules.json) file is crucial for us.
Every new release may break a build and can be hard for developers to understand what happens.

Current tests are runned only on next versions not on range edge versions which mean we do not cover the current range configuration.

**What is the chosen solution to this problem?**

* Fix the tests to be runned
* Move them to a dedicated file
* use a dedicated test command for it because thoses takes 2 minutes to run with cache and can be 30 minutes without cache.
* create a cron workflow that will create an issue if test fails

Screenshot 2022-04-08 at 16 56 41
...
Screenshot 2022-04-08 at 16 56 51

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: .github/workflows/tests-cron.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9 * * 1-5" # At 09:00 AM, Monday through Friday
+
+jobs:
+  tests:
+    name: Long running Tests
+    runs-on: ubuntu-latest
+    continue-on-error: true # we want the issue to popup even if sth goes wrong
```
Comment (smouillour) [inline .github/workflows/tests-cron.yml:12]: why do you need this?
You already added the if condition to manage the case of failure. https://github.com/Talend/ui/pull/3871/files#diff-d3782acde429bd2a63f81d47345c3de6f798bad3d04f757d2813bb77ece283a4R32

Comment (jmfrancois) [inline .github/workflows/tests-cron.yml:12]:
```suggestion
```

Comment (jmfrancois) [reply .github/workflows/tests-cron.yml]: indeed, let's remove it

**Review by smouillour — APPROVED** [review]

**Review by jsomsanith-tlnd — APPROVED** [review]
Thank you for that, I got a test fail twice on PRs because of that, but it wasn't related to my changes :D

**Pull request merged**